act/grids.md

```diff
@@ -0,0 +1,52 @@
+# Grids
+
+## Rule Identifier
+
+gel-components-grids
+
+## Rule Description
+
+Grids enable consistent and efficient visual layouts.
+
+## Rule Type
+
+atomic
+
+## Accessibility Requirements Mapping
+
+- [BBC GEL Grids](https://bbc.github.io/gel/foundations/grids/)
+  - Required for conformance to BBC GEL
+  - Outcome mapping:
+    - Any failed outcomes: success criterion is not satisfied
+    - All passed outcomes: success criterion needs further testing
+    - An inapplicable outcome: success criterion needs further testing
+
+## Input Aspects
+
+* DOM Tree
+
+## Applicability
+
+This rule applies to any HTML page that includes a BBC GEL Grids.
+
+## Expectations
+
+### Expectation 1
+
+Table elements must not be allowed to wrap and find themselves falling under different headers.
```

Comment:
It's not obvious to me why the Table element is mentioned here. We have another guide regarding data tables, should this expectation go there?